source/Octopus.Tentacle/Configuration/LogInitializer.cs

```diff
@@ -40,6 +40,9 @@ void InitializeLogs()
 
                 OctopusLogsDirectoryRenderer.History.Add(logDirectory);
                 OctopusLogsDirectoryRenderer.LogsDirectory = logDirectory;
+                
+                // Reload the configuration so NLog picks up the change immediately 
+                NLog.LogManager.Configuration = NLog.LogManager.Configuration.Reload();
```

Comment:
I didn't notice any issue when running the standard tentacle locally but if there is any concern I can wrap this in a check if tentacle is running as the k8s agent

Reply:
I think it should be fine 👍